### PR TITLE
add item identification for logs and planks

### PR DESCRIPTION
**Pull request by asdftemp**
I added item identification for logs and planks. Let me know if I need to change something. thanks

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java [hunk 1]
```diff
@@ -69,6 +69,49 @@
 	SITO_FOIL(Type.HERB, "Sito", "SF", ItemID.SITO_FOIL, ItemID.GRIMY_SITO_FOIL),
 	SNAKE_WEED(Type.HERB, "Snake", "SW", ItemID.SNAKE_WEED, ItemID.GRIMY_SNAKE_WEED),
 	VOLENCIA_MOSS(Type.HERB, "Volenc", "V", ItemID.VOLENCIA_MOSS, ItemID.GRIMY_VOLENCIA_MOSS),
+
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java:72]: Remove this extra line
```suggestion
```

File: runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java [hunk 2]
```diff
@@ -69,6 +69,49 @@
...
+	MAGIC_LOG(Type.LOGS, "Magic", "MAG", ItemID.MAGIC_LOGS),
+	REDWOOD_LOG(Type.LOGS, "Red", "RED", ItemID.REDWOOD_LOGS),
+
+	PYRE_LOGS(Type.LOGS, "Pyre", "P", ItemID.PYRE_LOGS),
+	ARCTIC_PYRE_LOGS(Type.LOGS, "Artic  P", "AP", ItemID.ARCTIC_PYRE_LOGS),
+	OAK_PYRE_LOGS(Type.LOGS, "Oak P", "OAKP", ItemID.OAK_PYRE_LOGS),
+	WILLOW_PYRE_LOGS(Type.LOGS, "Willow P", "WILP", ItemID.WILLOW_PYRE_LOGS),
+	TEAK_PYRE_LOGS(Type.LOGS, "Teak P", "TEAKP", ItemID.TEAK_PYRE_LOGS),
+	MAPLE_PYRE_LOGS(Type.LOGS, "Maple P", "MAPP", ItemID.MAPLE_PYRE_LOGS),
+	MAHOGANY_PYRE_LOGS(Type.LOGS, "Mahogany P", "MAHP", ItemID.MAHOGANY_PYRE_LOGS),
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java:103]: Does this overlap the item next to it? "Mahogany P" seems pretty long

Comment (asdftemp) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java]: Yes this did overlap. found a few more and also fixed those

**Comment by asdftemp** [comment]
I made a mistake committing the first time, hence the force push. let me know if i should recreate the pr. thanks

**Pull request merged**